Wrappers/Python/cil/optimisation/algorithms/PDHG.py

```diff
@@ -339,7 +340,10 @@ def set_up(self, f, g, operator, tau=None, sigma=None, initial=None, **kwargs):
         self.g = g
         self.operator = operator
 
-        self.set_step_sizes(sigma=sigma, tau=tau) 
+        self.set_step_sizes(sigma=sigma, tau=tau)
+
+        if kwargs.get('check_convergence', True):
```

Comment:
I'd rather not have it in the `kwargs`. Actually the whole creator signature should clearly expose the parameter used to instantiate the base `Algorithm` class, i.e. `max_iteration` ,`update_objective_interval` and `log_file`.

Reply:
We can live with it, provided that we will fix the whole `kwargs` business #1115